lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -10,7 +10,14 @@
 
 import matplotlib as mpl
 from matplotlib import dviread, pyplot as plt, checkdep_usetex, rcParams
-from matplotlib.backends.backend_pdf import PdfPages
+from matplotlib.cbook import _get_data_path
+from matplotlib.ft2font import FT2Font
+
+# Weird py23 deprecation warning from fonttools
+with pytest.warns(DeprecationWarning):
```

Comment:
I think this is an assertion that a warning is raised, which will probably break when a future version of fonttools fixes the code that causes the warning. A suitable [filterwarnings setting](https://docs.pytest.org/en/6.2.x/warnings.html#deprecationwarning-and-pendingdeprecationwarning) in pytest.ini would let us ignore fonttools specific warnings.

Reply:
Even after adding this to `pytest.ini`, this failed.
However, adding it to `lib/matplotlib/testing/conftest/py` worked.